outputs/client.go

```diff
@@ -138,6 +138,12 @@ func (c *Client) Post(payload interface{}) error {
 		req.Header.Add("event-namespace", c.Config.Kubeless.Namespace)
 	}
 
+	if c.OutputType == Openfaas {
+		req.Header.Add("event-id", uuid.New().String())
+		req.Header.Add("event-type", "falco")
+		req.Header.Add("event-namespace", c.Config.Openfaas.FunctionNamespace)
+	}
+
```

Comment:
I don't think these headers are mandatory for OpenFaas as they are for Kubeless. If you want to set them, maybe we could merge with Kubeless condition.

```go
	if c.OutputType == Kubeless || c.OutputType == Openfaas {
		req.Header.Add("event-id", uuid.New().String())
		req.Header.Add("event-type", "falco")
        switch c.OutputType {
		case Kubeless:
			req.Header.Add("event-namespace", c.Config.Kubeless.Namespace)
		case Openfass:
			req.Header.Add("event-namespace", c.Config.Openfaas.FunctionNamespace)	
		}
	}
```

Reply:
yeah, these are not mandatory for OpenFaaS, let me clean this up 👍